.travis.yml

```diff
@@ -1,40 +1,45 @@
-# The only current way of using OS X virtual machines in Travis CI
-# is setting up a generic environment
-# This is not a problem because Python will be installed through conda
-
-language: generic
-os: osx
```

Comment:
Beware! We were using Travis CI to test things on OS X. For Linux, it's Circle CI, and perhaps you'll need to touch `appveyor.yml` as well.